artview/components/plot_radar.py

```diff
@@ -598,14 +599,46 @@ def getPathInteriorValues(self, path):
         from .tools import interior_radar
         radar = self.Vradar.value
         if radar is None:
-            return (np.array([]),)*7
+            return None
 
         xy, idx = interior_radar(path, radar, self.Vtilt.value)
-        aux = (xy[:, 0], xy[:, 1], radar.azimuth['data'][idx[:, 0]],
-               radar.range['data'][idx[:, 1]] / 1000.,
-               radar.fields[self.Vfield.value]['data'][idx[:, 0], idx[:, 1]],
-               idx[:, 0], idx[:, 1])
-        return aux
+        xaxis = {'data':  xy[:, 0] * 1000.,
+                 'long_name': 'X-coordinate in Cartesian system',
+                 'axis': 'X',
+                 'units': 'm'}
+
+        yaxis = {'data':  xy[:, 1] * 1000.,
+                 'long_name': 'Y-coordinate in Cartesian system',
+                 'axis': 'Y',
+                 'units': 'm'}
+
+        azi = radar.azimuth.copy()
+        azi['data'] = radar.azimuth['data'][idx[:, 0]]
+
+        rng = radar.range.copy()
+        rng['data'] = radar.range['data'][idx[:, 1]]
+
+        field = radar.fields[self.Vfield.value].copy()
+        field['data'] = radar.fields[self.Vfield.value]['data'][
+            idx[:, 0], idx[:, 1]]
+
+        ray_idx = {'data': idx[:, 0],
+                   'long_name': 'index in ray dimension'}
+        rng_idx = {'data': idx[:, 1],
+                   'long_name': 'index in range dimension'}
+
+        axes = {'x_disp':xaxis,
+                'y_disp':yaxis,
+                'ray_index':ray_idx,
+                'range_index':rng_idx,
+                'azimuth':azi,
+                'range':rng}
```

Comment:
I wonder if we should include the units in a couple of these because they are different than the display which makes the adjustment automatically (e.g. meters to kilometers):
```
        axes = {'x_disp_m':xaxis,
                'y_disp_m':yaxis,
                'ray_index':ray_idx,
                'range_index':rng_idx,
                'azimuth':azi,
                'range_m':rng}```